docs/docs_zh/sources/source/conf.py

```diff
@@ -86,13 +106,15 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-#
+
+from package_info import __version__
+
 # The short X.Y version.
 # version = "0.10.0"
-version = str(nemo.__version__)
+version = __version__
 # The full version, including alpha/beta/rc tags.
 # release = "0.9.0"
-release = str(nemo.__version__)
+release = __version__
```

Comment:
Fascinating... I thought I have introduced _exactly_ that change in one of my PRs... :)

Reply:
ehehehe
https://github.com/NVIDIA/NeMo/blob/64eed9a5980b7f4eeca250e7211d7672fd8fe35c/docs/source/conf.py#L96

yap, I did ;)

Reply:
(but in un-merged branch that waits for RTD to FINALLY WORK... :] )